neuro_san/http_sidecar/logging/http_logger.py

```diff
@@ -87,10 +87,13 @@ def setup_logging(self):
         """
         Setup logging from configuration file.
         """
+        # Need to initialize the forwarded metadata default values before our first
+        # call to a logger.
         current_dir: str = pathlib.Path(__file__).parent.parent.resolve()
         setup_logging(HttpLogger.HTTP_LOGGER_NAME, current_dir,
                       'AGENT_SERVICE_LOG_JSON',
-                      'AGENT_SERVICE_LOG_LEVEL')
+                      'AGENT_SERVICE_LOG_LEVEL',
+                      self.base_metadata)
 
```

Comment:
Initialize thread-specific logger record fields. They are not likely to contain anything non-default,
but at least logger will not crash.